expo/App.js

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from "react";
+import { Pressable, StyleSheet, Text, View } from "react-native";
+
+import * as SQLite from "expo-sqlite";
+
+import { electrify } from "electric-sql/dist/drivers/expo-sqlite";
```

Comment:
We should use the shorter import paths defined in the exports mapping in package.json. So `import { ... } from 'electric-sql/expo'` in this case.